app/assets/stylesheets/blogs.scss

```diff
@@ -34,4 +34,12 @@
     border-radius: 5px;
     border: solid 1px gray;
   }
+
+  .js-smart-uploader {
+    transition: box-shadow 0.2s linear;
+
+    &.dragover {
+      box-shadow: #c9ff00 0 0 3px !important;
```

Comment:
Color literals like `#c9ff00` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used